devicetypes/smartthings/aeon-multisensor-gen5.src/aeon-multisensor-gen5.groovy

```diff
@@ -12,14 +12,17 @@
  *
  */
 metadata {
-	definition (name: "Aeon Multisensor Gen5", namespace: "smartthings", author: "SmartThings") {
+	definition (name: "Aeon Multisensor Gen5", namespace: "smartthings", author: "SmartThings", category: "C3") {
 		capability "Motion Sensor"
 		capability "Temperature Measurement"
 		capability "Relative Humidity Measurement"
 		capability "Illuminance Measurement"
 		capability "Configuration"
 		capability "Sensor"
 		capability "Battery"
+		capability "Health Check"
+
+		command "configureAfterSecure"
```

Comment:
The `Aeon MultiSensor Gen5` valueTile `configureAfterSecure` is the command on the tile. Need to add this or else that configure tile doesn't actually call `configure`. 
@mckeed - is this the right approach?